bin/merge.sh

```diff
@@ -64,12 +66,12 @@ merge_cv32e40s_into_cv32e40x-dv () {
 
 move_files_40s_into_40x () {
 
-  echo "=== Exchange 40x/X with 40s/S in file names ==="
+  echo "=== Replace 40s/S with 40x/S in file names ==="
```

Comment:
Is it supposed to be Replace 40s/S with 40x/**X** ?

Reply:
Fixed.

Reply:
RESOLVED